.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,9 +22,11 @@ Any special considerations for deployment? Consider both:
 
 ## Checklist
 
-### If you made changes to the app code:
+### If you made changes to the code:
 
 - [ ] Unit and functional tests pass on the development VM
+- [ ] Added Python type annotations/hints in the Python code
+- [ ] **make lint** passes for the updated code.
```

Comment:
As discussed during our chat today, I think a nice approach to adding type annotations to our codebase would be to begin by encouraging people (but not _requiring_) their addition. Now that we have this nice page of documentation, developers can get accustomed to them, and at a later date we can enforce a requirement. 

That said, I propose we leave out these pull request template changes for now (note the linting comment is in #3113). What do you think?

Reply:
@redshiftzero I will remove the PR template change as it is already taken care of by 3113.